hummingbot/core/utils/trading_pair_fetcher.py

```diff
@@ -42,7 +42,7 @@ def _fetch_pairs_from_connector_setting(
     async def fetch_all(self, client_config_map: ClientConfigAdapter):
         connector_settings = self._all_connector_settings()
         for conn_setting in connector_settings.values():
-            # XXX(martin_kou): Some connectors, e.g. uniswap v3, aren't completed yet. Ignore if you can't find the
+            # XXX(martin_kou): Some connectors, e.g. amm v3, aren't completed yet. Ignore if you can't find the
```

Comment:
We have connector / exchange named uniswap
https://hummingbot.org/exchanges/uniswap/

this changes here not needed